rest-testing/pom.xml

```diff
@@ -150,6 +163,7 @@
         <rest-assured.version>2.9.0</rest-assured.version>
         <cucumber.version>1.2.5</cucumber.version>
         <wiremock.version>2.4.1</wiremock.version>
+        <karate.version>0.6.0</karate.version>
```

Comment:
I see that 0.6.1 has just come out earlier this month. Let's go ahead and update the version.